custom_components/localtuya/common.py

```diff
@@ -209,13 +209,19 @@ def __init__(self, device, config_entry, dp_id, logger, **kwargs):
         self._dp_id = dp_id
         self._status = {}
         self.set_logger(logger, self._config_entry.data[CONF_DEVICE_ID])
+        self._stored_state = {}
 
     async def async_added_to_hass(self):
         """Subscribe localtuya events."""
         await super().async_added_to_hass()
 
         self.debug("Adding %s with configuration: %s", self.entity_id, self._config)
 
+        state = await self.async_get_last_state()
+        if state:
+            self._stored_state = state
```

Comment:
Do we really have to store it in `self._stored_state`, can't we just pass the value as an argument to `status_restored`?